src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null,
+};
+
+export class App extends React.Component {
+  state: Readonly<State> = {
+    pressedKey: null,
+  };
+
+  componentDidMount(): void {
```

Comment:
Unnecessary type, React & typescript already know about this
```suggestion
  componentDidMount() {
```